lib/assets.js

```diff
@@ -48,7 +49,26 @@ module.exports = {
 
     index: function (server) {
         return function (req, res, next) {
-            req.pipe(filed(path.join(server.root, 'views/index.html'))).pipe(res);
+            fs.readFile(path.join(server.root, 'views/index.html'), 'utf8', function (err, data) {
+                if (imageVersion) {
+                    data = data.replace(/_IMAGE_VERSION_/g, 'image version: ' + imageVersion);
+                }
+                var done = function (error, info) {
+                    var result = data;
+                    if (error) {
+                        req.log.debug('Error getting git information', error);
```

Comment:
Why is the error message referring to "git information"? This is the build image-version info yes?